lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -329,10 +327,8 @@ changeUTxO
     -> Set Tx
     -> s
     -> UTxO
-changeUTxO proxy pending = evalState $ do
-    ourUtxo <- mapM (state . utxoOurs proxy) (Set.toList pending)
-    let ins = txIns pending
-    return $ fold ourUtxo `restrictedBy` ins
```

Comment:
Not even sure why this was even there... The spec says:

![image](https://user-images.githubusercontent.com/5680256/58077467-dfbfef80-7bac-11e9-95c6-a6aafbd4ec2d.png)

Which is exactly what the first line already does... The two others are non-sense.